src/util/util.shared.ts

```diff
@@ -547,4 +552,28 @@ export const Utils = {
     }
     return rpc?.nodes?.[0].url || `${Constant.TATUM_API_URL.V3}blockchain/node/${network}`.concat(path || '')
   },
+  addQueryParams: (basePath: string, queryParams?: Record<string, string | string[]>): string => {
```

Comment:
Formatting in this file seems off, lines are not aligned / indented correctly.

Reply:
<img width="1285" alt="image" src="https://github.com/tatumio/tatum-js/assets/10884869/7ecac6f5-3979-4843-97b4-68b979be7fa4">